.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,33 @@
+name: CI - To-Do app
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+      - 'Dockerfile'
+      - '.github/**'
+      - '.dockerignore'
+      - 'docker-compose.yml'
+      - 'ansible/**'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - 'Dockerfile'
+      - '.github/**'
+      - '.dockerignore'
+      - 'docker-compose.yml'
+      - 'ansible/**'
```

Comment:
Nice comprehensive list of paths to ignore 👍

Reply:
Thanks Jack